File: pset6/mario.py
import cs50

def main():
    print("Height: ",end="")
    height = cs50.get_int()
    while height < 0 or height > 23:
        print("Height: ",end="")
        height = cs50.get_int()
    
    
    for i in range(1,height+1):
        for j in range(height - i,0,-1):
            print(" ",end = "")
        for k in range(i+1,0,-1):
            print("#",end = "")
        
        print()


if __name__ == "__main__":
    main()
            
        
    
    

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    printf("Height: ");
    int height = get_int();
    while(height < 0 || height > 23)
    {
        printf("Height: ");
        height = get_int();
    }
    
    //Outer Loop for Number of Lines (Height)
    for(int i = 1;i <= height; i++){
        //Printing Spaces
        for(int j = height - i;j > 0; j--)
            printf(" ");
        //Printing Hash
        for(int k = i + 1; k > 0; k--)
            printf("#");
        
        printf("\n");
    }
    
}